package.json

```diff
@@ -131,7 +131,7 @@
     "react-native-device-info": "^10.3.0",
     "react-native-fast-image": "^8.6.0",
     "react-native-pager-view": "^5.4.25",
-    "react-native-reanimated": "2.14.4",
+    "react-native-reanimated": "3.5.4",
```

Comment:
@mohitharshan123 We also need to update in peer dependency 
https://github.com/bigbinary/neeto-ui-rn/blob/cd9f5566be0a236db3c8870c04347d6bc58d2e04/package.json#L48